packages/compiler/core/parser.ts

```diff
@@ -468,7 +480,7 @@ export function parse(code: string | SourceFile, options: ParseOptions = {}): Ca
     decorators: DecoratorExpressionNode[]
   ): NamespaceStatementNode {
     parseExpected(Token.NamespaceKeyword);
-    let currentName = parseIdentifierOrMemberExpression();
+    let currentName = parseIdentifierOrMemberExpression(undefined, true);
```

Comment:
I don't see the change making these args required. `(undefined, true)` doesn't really help the reader here IMHO.

Reply:
Feel like if we want to force the args to be passed maybe having named options for readability would be better. Have to dig into the declaration right now to know what the boolean mean

Reply:
Ahh sorry this is a bit of code left over from the next PR which adds support for `::` references in regular cadl. That function grows an additional boolean parameter, but I agree three booleans is getting a little ridiculous. I plan to split out the different use cases into individual functions and call in to parseIdentifierOrMemberExpression from there.